src/feedback.tsx

```diff
@@ -369,7 +370,7 @@ export const InfoFeedbackScreen = ({
                 testid: 'InfoFeedbackScreen',
                 ...dataAttributes,
             }}
-            asset={<Icon size="100%" />}
+            asset={<Icon size="100%" color={vars.colors.brand} />}
```

Comment:
I took the opportunity to fix this bug because when you use a custom icon, the icons is not showing with brand color
https://tinyurl.com/26ky4g44